### PR TITLE
Solved the exercise.

### PR DESCRIPTION
**Pull request by fadamDS**

### CODE REVIEW

**Comment by fadamDS** [comment]
Updated the exercise for parsing all pages.

File: exercises.ipynb [hunk 1]
```diff
@@ -54,40 +51,935 @@
...
+   "outputs": [],
+   "source": [
+    "def parse_all_bricks(start_url):\n",
+    "    \n",
+    "    link = start_url\n",
+    "    \n",
+    "    # Setting up the starting link\n",
+    "    current_link = start_url\n",
+    "    \n",
+    "    # Setting up list with one entry, easier to concatonate\n",
```
Comment (nandanrao) [inline exercises.ipynb:102]: ??? You can concatenate an empty list just fine!

```
[] + ['foo']
```

File: exercises.ipynb [hunk 2]
```diff
@@ -54,40 +51,935 @@
...
+    "    return titles\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 101,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def parse_all_bricks(start_url):\n",
```
Comment (nandanrao) [inline exercises.ipynb:95]: Nice job putting it all into a function!

**Review by nandanrao — COMMENTED** [review]
Great!

**Pull request closed without merging**